docker-compose.yml

```diff
@@ -31,7 +31,7 @@ services:
     restart: ${RESTART_POLICY}
     security_opt:
       - no-new-privileges:true
-    pids_limit: 200
+    pids_limit: 400
```

Comment:
More than 200 processes means that something is very wrong with the system. I'd like to get a list of the processes if possible and see if there's an actual bug from the product that needs to be fixed. Otherwise, there's a chance that we are hiding something under the rug here.